app/src/marketplace/components/Modal/ChooseAccessPeriodDialog/CurrencySelector/index.jsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React from 'react'
+import cx from 'classnames'
+
+import type { PaymentCurrency } from '$shared/flowtype/common-types'
+import SvgIcon from '$shared/components/SvgIcon'
+
+import styles from './currencySelector.pcss'
+
+type Props = {
+    onChange: (currency: PaymentCurrency) => void,
+    paymentCurrency: PaymentCurrency,
+}
+
+const availableCurrencies = process.env.NEW_MP_CONTRACT ? ['DATA', 'ETH', 'DAI'] : ['DATA']
```

Comment:
We need to leverage the `NEW_MP_CONTRACT` switch here so this code can land in production without the deployed mainnet uniswap contract, and MP2 contract upgrade (I'm unsure when that will land, maybe later this month or next).

`availableCurrencies` will offer DATA as the only option until we're ready.

Reply:
Sure, though I meant you can still use here the constant value (instead of written value):
```
const availableCurrencies = process.env.NEW_MP_CONTRACT ? [
  paymentCurrencies.DATA, 
  paymentCurrencies.ETH, 
  paymentCurrencies.DAI,
] : [
  paymentCurrencies.DATA,
]
```

Reply:
Ah, silly me, ofc!